CONTRIBUTING.md

```diff
@@ -46,7 +46,7 @@ As an informative document, bugs in the meta-schema may be fixed in place to ali
 
 ## Conduct
 
-All official channels including the mailing list, GitHub organization, and Freenode channel, follow the IETF Guidelines for Conduct as specified in [RFC7154](https://tools.ietf.org/html/rfc7154).
+All official channels including the mailing list, GitHub organization, Slack server, and Freenode channel, follow our [Code of Conduct](https://github.com/json-schema-org/.github/blob/main/CODE_OF_CONDUCT.md).
```

Comment:
Do we still have a Freenode channel? I assume we don't have an IRC channel at all at this point and if we did it's probably on Libera (but I haven't checked in, maybe I should)

Reply:
There's a couple of users in the Libera channel looks like, including @karenetheridge. (I don't have access to Freenode anymore nor do I think anyone else does, they password reset everyone during the hostile takeover or something right?)

TL;DR maybe it's easy to avoid all my comments by changing this to say IRC channels and remove the reference to Freenode.

Other than that, lgtm.

Reply:
@Julian thanks, I changed it to "IRC channels".